### PR TITLE
Modify template to use `react-native-tvos` for TV support

### PR DESCRIPTION
**Pull request by douglowder**
Make changes to support TV:

- Change name of this template repo
- Use the `react-native-tvos` package in template `package.json`
- Bring in Podfile and Xcode project from `react-native-tvos`
- Modify `App.tsx` to remove status bar and add TV specific example code

### CODE REVIEW

File: template/App.tsx [hunk 1]
```diff
@@ -26,28 +27,35 @@ import {
   ReloadInstructions,
 } from 'react-native/Libraries/NewAppScreen';
 
+import 'react-native/tvos-types.d';
```
Comment (emin93) [inline template/App.tsx:30]: Is this required? I think it should work out of the box because typings are being included automatically.

**Review by emin93 — COMMENTED** [review]
Maybe also adjust the `README.md` to show the correct usage for tvOS?

**Comment by emin93** [comment]
👋 @dlowder-salesforce 

Thanks for the effort, looks good! I left two comments, but other than that I didn't spot anything.

**Review by emin93 — CHANGES_REQUESTED** [review]
👋 @dlowder-salesforce

Thanks for the effort, looks good! I left two comments, but other than that I didn't spot anything.

File: template/App.tsx (hunk 1 above)
Comment (douglowder) [reply template/App.tsx:30]: That is needed to pull in some types that only exist in the TV repo (focus management etc.)

**Pull request merged**